ark/repo/tsup.config.ts

```diff
@@ -2,7 +2,7 @@ import { defineConfig } from "tsup"
 
 export default defineConfig({
 	entry: ["main.ts"],
-	format: ["cjs"],
+	format: ["cjs", "esm"],
```

Comment:
Sorry about this, it was just in this state temporarily as I was trying to get a CJS compatible build for `typescript-runtime-type-benchmarks` (fine to leave this change in).